\documentclass[12pt,a4paper]{article}
\usepackage{inverba}

\newcommand{\userName}{Cullyn Newman} 
\newcommand{\class}{BI 320} 
\newcommand{\institution}{Portland State University} 
\newcommand{\theTitle}{\color{g-Leaf} Organismal Physiology}

\begin{document}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\tableofcontents
\cleardoublepage
\fancyhead{}
\fancyhead[R]{\hyperlink{home}{\nouppercase\leftmark}}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\clearpage
\fancyhead[L]{Week 1}
%\begingroup
%%%%%%%%%%%%%%%%%%%%%%%%%%%%% Chapter 1 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%\begingroup
\clearpage
\section{Animals and Environments}\phantomsection
\subsection{Introduction}
\begin{itemize}
    \item What is physiology?
        \begin{itemize}
            \item Form and function of organisms; the study of how organisms work.
        \end{itemize}
    \item Central questions of physiology: {\color{o-Sun}mechanism} and {\color{o-Sun}origin}.
        \begin{itemize}
            \item Mechanism:
                \begin{itemize}
                    \item Refers to the {\color{o-Sun}components} of living organisms and understanding {\color{o-Sun}how} components interact to enable the organism to function.
                \end{itemize}
            \item Origin:
                \begin{itemize}
                    \item Asks why a mechanism exists, or {\color{o-Sun}what} is the mechanistic {\color{o-Sun}adaptive significance} of the mechanism.
                \end{itemize}
            \item Mechanism and adaptive significance are distinct concepts; knowing about one doesn't necessarily mean you know anything about the other.
        \end{itemize}
    \item Krogh's principle: \begin{quote}\color{G-Moon}
        "For such a large number of problems there will be some animal of choice or a few such animals on which it can be most conveniently studied."\end{quote}
        \begin{itemize}
            \item This idea is central to disciplines that rely on the \textit{comparative method}.
            \item Other key concepts:
                \begin{itemize}
                    \item There is unity in diversity; many organisms are very much alike at the most fundamental levels. 
                    \item The differences are subject to particular niches and often highly specialized that allow for biologist to study more complex systems.
                    \item The similarities allow us overcome technical limitations via animals that are easier to study.
                \end{itemize}
        \end{itemize}
    \item Physiology subdisciplines: 
        \begin{itemize}
            \item Mechanistic: emphasizes the mechanisms by which organisms perform their life functions.
            \item Evolutionary: emphasizes evolutionary origins and the adaptive significance of traits.
            \item Comparative: emphasizes the way in which diverse phylogenetic groups resemble and differ from each other.
            \item Environmental: emphasizes the ways in which physiology and ecology interact.
            \item Integrative: emphasizes the importance of all levels of organization, from genes to proteins and tissues to organs in order to better understand whole physiological systems.
        \end{itemize}
\end{itemize}

\subsection{Homeostasis}
\begin{itemize}
    \item Important ideas to remember:
        \begin{itemize}
            \item Organisms are structurally dynamic; form stays relatively static while individual cells recycle frequently.
            \item Most cells are exposed to the {\color{o-Sun}internal} environment, not external.
            \item Internal cells may vary or kept constant with the environment.
        \end{itemize}
    \item Temperature regulation:
        \begin{itemize}
            \item \textbf{Conformity}: organism's internal temperature {\color{o-Sun}correlates} with external temperature in a particular range of temperatures. 
            \item \textbf{Regulation}: internal environment is held mostly {\color{o-Sun}constant} using celluar mechanisms.
        \end{itemize}
    \item \textbf{Homeostasis}: the coordinated physiological processes that maintain a relatively constant state in the organism.
        \begin{itemize}
            \item {\color{pos}Positive feedback}: less common in homeostasis due difficulty in regulation; leads to runaway effect easily.
            \item {\color{neg}Negative feedback}: more common in homeostasis due to self correcting nature.
            \item \textbf{Effector}: executes the change in action that produces an effect, e.g. signals to increase temperature.
            \item \textbf{Sensor}: sense changes in environment and sends information to the effector.
        \end{itemize}
\end{itemize}

\subsection{Physiology and Time}
\begin{itemize}
    \item Timeframes of physiological change:
        \begin{itemize}
            \item \textbf{Acute}: short-term, reversible, and quick to adapt to changes in environment. Usually minutes to hours.
            \item \textbf{Chronic}: long-term after prolonged exposure to new environments. Changes are usually reversible, but often slower. 
                \begin{itemize}
                    \item Chronic can be termed acclimation, or phenotypic plasticity/flexibility.
                     \item Repetitive acute responses usually lead to chronic responses.
                \end{itemize}
            \item \textbf{Evolutionary}: changes due to alteration in gene frequencies in {\color{o-Sun}populations} exposed to new environments.
        \end{itemize}
    \item Acclimation is {\color{false}not the same} as adaption.
        \begin{itemize}
            \item \textit{Adaption} is an evolutionary trait present at high frequency in a population due to survival/reproductive advantages. 
            \item Not all traits are adaptations.
            \item The amount of natural variation in a trait must be considered across populations, species etc.
        \end{itemize}
\end{itemize}
%\endgroup
%%%%%%%%%%%%%%%%%%%%%%%%%%%%% Chapter 1 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%%%%%%%%%%%%%%%%%%%%%%%%%%%%% Chapter 2 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%\begingroup
\clearpage
\section{Molecules and Cells in Animal Physiology}\phantomsection
\subsection{Cell Membrane Review}
\begin{itemize}
    \item Major cell memberane structures:
        \begin{itemize}
            \item \textbf{Glycoproteins}: carbohydrate chain attached to a protein.
            \item \textbf{Glycolipids}: similar to glycoproteins, but attached to lipid molecues.
            \item \textit{Glycocalyx}: combination of glycoproteins and glycolipids on the surface of cell.
            \item \textbf{Integral proteins}: embedded in phospholipid bilayer.
            \item \textbf{Peripheral proteins}: associated with one side of the bilayer.
        \end{itemize}
    \item \textbf{Unsaturated phospholipid}: whey hydrocarbon tails contain double bonds (less hydrogen).
        \begin{itemize}
            \item Increase membrane fluidity due to extra space created.
        \end{itemize}
    \item The fluidity of the cell membrane allows proteins to from complexes and dynamically change shape.
\end{itemize}

\subsection{Enzyme Fundamentals}
\begin{itemize}
    \item \textbf{Enzymes}: a protein catalyst that plays two primary roles: {\color{o-Sun}accelerating} and {\color{o-Sun}regulating} chemical reactions. 
    \item \textit{Substrates}: the initial reactants of the reaction that an enzyme catalyzes.
    \item \textbf{Enzyme-substrate-complex (E-S)}: a combination of enzyme (E) with a molecule of substrate (S) that starts a reaction.
        \begin{itemize}
            \item Usually stabalized by {\color{o-Sun}non-covalent} bonds.
            \item The substrate is converted to a product by first becomeing an \textit{enzyme-product complex (E-P)}, then dissociates to yield free product and free enzyme.
            \item {\color{o-Sun}\(E+S\rightleftharpoons \text{E-S} \rightleftharpoons \text{E-P} \rightleftharpoons E + P\)}
        \end{itemize}
    \item \textbf{Saturation kinetics}:
        \begin{itemize}
            \item \textbf{V\(_{\text{max}}\)}: the maximum velocity of a reaction and is determined by:
                \begin{itemize}
                    \item The {\color{o-Sun}number} of active enzyme molecues present relative to substrate.
                    \item The catalytic {\color{o-Sun}effectiveness} of each enzyme molecule.
                    \item These properties usually undergo heavy selection pressure.
                \end{itemize}
            \item \textit{Saturated}: all enzymes are occupied by a substrate molecule nearly all the time and now unable to increase reaction velocity.
            \item \textbf{Hyperbolic}: asymptotically approaches V\(_{\text{max}}\)
                \begin{itemize}
                    \item Tends to happen when enzymes have just one substrate binding site.
                    \item Or when substrate sites behave independently
                \end{itemize}
            \item \textbf{Sigmodal}: approaches V\(_{\text{max}}\) with a sigmodal trajectory.
                \begin{itemize}
                    \item When multiple sites influence each other.
                \end{itemize}
            \item \textbf{Turnover number (k\(_{\text{cat}}\))}: the {\color{o-Sun}total effectiveness}, expressed as the number of substrate molecules coverted to product per second by each enzyme molecule when saturated.
                \begin{itemize}
                    \item Depends partly on the \textit{activation energy} of the enzyme-catalyzed reaction.
                    \item \textbf{Activation energy}: the energy required for the substrate to enter the \textit{transition state}. 
                    \item \textbf{Transition state}: the intermediate chemical state between substrate and product.
                    \item Enzymes {\color{o-Sun}lower the activation energy} required to enter transition state.
                \end{itemize}
        \end{itemize}
    \item \textbf{Enzyme-substrate affinity}: The proclivity of the enzyme to form a complex with the substrate when they meet.
        \begin{itemize}
            \item {\color{pos}Likely} complex formation results in {\color{pos}high-affinity}.
            \item {\color{neg}Unlikely} complex formation results in {\color{neg}low-affinity}.
            \item Affinity affects the shape of the reaction velocity.
                \begin{itemize}
                    \item {\color{pos}Higher} affinity produces a {\color{pos}steeper} velocity, and a {\color{neg}lower} affinity produces a more {\color{neg}linear} result.
                    \item Enzyme concentration is not changed.
                \end{itemize}
            \item \textbf{Half-saturation constant, \(\mathbf{K_m}\)}: the substrate concentration required to attain one-half maximum reaction velocity.
                \begin{itemize}
                    \item \(K_m\) and enzyme-substrate affinity are  {\color{o-Sun}inversely related}.
                    \item i.e. {\color{neg}low-affinity} enzyme has a {\color{pos}greater \(K_{m}\)}.
                \end{itemize}
        \end{itemize}
    \item \textbf{Molecular Flexibility}:
        \begin{itemize}
            \item \textbf{Conformation}: the three-dimensional shape of a protein.
                \begin{itemize}
                    \item Stabalized mostly by {\color{o-Sun}weak, noncovalent bonds}---hydrogen, van der Waals, hydrophobic, electrostatic, etc.
                    \item Weak interactions allow for easy yet stable conformational changes.
                \end{itemize}
            \item Enzyme molecules composed of two, three or our proteins are called \textit{dimeric, trimeric}, or \textit{tetrameric} respectively.
        \end{itemize}
    \item Enzymes catalyze reversible reactions in both directions.
        \begin{itemize}
            \item This is because they accelrate the approach towards equilibrium (principles of mass action).
        \end{itemize}
    \item \textbf{Ligand}: any molecule that selectively binds by noncovalent bonds to structurally and complementary sites on a specific protein.
    \item \textbf{Cooperativity}: the interactions between multiple binding sites that may facilitate or inhibit the binding of other sites.
        \begin{itemize}
            \item Can either {\color{pos}positive} or {\color{neg}negative}; {\color{pos}facilitating} or {\color{neg}inhibiting} binding on the same molecule.
            \item \textit{{\color{o-Sun}Homotropic} cooperativity}: facilitation or inhibition of the {\color{o-Sun}same ligand}.
            \item \textit{{\color{o-Sun}Heterotropic} cooperativity}: influences on the binding of {\color{o-Sun}other ligands}.
            \item Interactions occur {\color{o-Sun}at a distance}, resulting in delayed, or rippling responses.
            \item \textbf{Allosteric modulation}: the modulation of the {\color{o-Sun}catalytic properties}.
                \begin{itemize}
                    \item \textbf{Allosteric sites}: nonsubstrate-binding regulatory sites for {\color {o-Sun}nonsubstrate ligands} that modulate the catalytic properties.
                    \item \textit{Allosteric modulators}: the nonsubstrate ligands.
                    \item Allosteric {\color{pos}\textbf{activation}:increases} and {\color{neg}\textbf{inhibition}:impairs} affinity, thus the {\color{o-Sun}catalytic activity}. 
                \end{itemize}
        \end{itemize}
    \item \textbf{Isozymes}: enzymes that catalyze the same chemical reaction but differ in amino acid sequence.
    \item \textbf{Interspecific enzyme homologs}: different molecular forms of an enzyme coded by homologous gen loci in different species.
        \begin{itemize}
            \item Isozymes and interspecific enzyme homologs often {\color{o-Sun}differ} in their {\color{o-Sun}catalytic} and {\color{o-Sun}regulatory} properties.
            \item Functional differences often prove to be adaptive in different environments.
        \end{itemize}
\end{itemize}
%\endgroup
%%%%%%%%%%%%%%%%%%%%%%%%%%%%% Chapter 2 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%\endgroup

\clearpage
\fancyhead[L]{Week 2}
%\begingroup
%\endgroup
%%%%%%%%%%%%%%%%%%%%%%%%%%%%% Chapter 3 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%\begingroup
\clearpage
\section{Genomics}\phantomsection
\subsection{Genomics}
\begin{itemize}
    \item \textbf{Genomics}: study of the genomes---the full set of genetic material---of organisms.
    \item Metods of genomics:
        \begin{itemize}
            \item Computational biology and bioinformatics use various computational methods to process large amount of genomic data.
            \item \textbf{High-throughput}: methods of analyzing large data with out much human attention and mostly computation.
            \item \textbf{Annotation}: laborious direct human interpretation.
        \end{itemize}
    \item The {\color{o-Sun}overarching goals} of genomics is to elucidate the {\color{o-Sun}evolution} and the {\color{o-Sun}current functioning} of genes and genomes.
    \item \textbf{Gene families}: genes that share distinctive DNA base sequences and \textit{tend} to code for functionally similar proteins.
    \item \textbf{Postgenomic era}: the study of species after genome is sequenced.
\end{itemize}

\subsection{Transcriptomics}
\begin{itemize}
    \item \textbf{Transcriptomics}: the study of which genes are transcribed to make mRNA and the rates at which they are transcribed.
        \begin{itemize}
            \item aka transcription profiling.
            \item Implies the 
            study of great numbers of mRNAs.
        \end{itemize}
    \item \textbf{Transcriptome}: a species full set set of mRNA molecules. It represents the full complement of genes being transcribed at any given time.
        \begin{itemize}
            \item Time is emphasized; it's a snapshot transcription activity during the observed period.
            \item Very useful in comparative methods.
        \end{itemize}
    \item Methods of transcriptomics:
        \begin{itemize}
            \item \textbf{DNA midroarrays}: aka gene chips; a high throughput method tht allows simultaneous analysis of large number of mRNAs.
            \item \textbf{mRNA sequencing}: aka RNA-Seq; similar to microarrays, but can identify both known and {\color{o-Sun}novel} transcripts.
                \begin{itemize}
                    \item More sensitive than microarrays.
                    \item Readily applicable across wide range of species.
                \end{itemize}
            \item \textbf{Gene manipulation}: studies that permit the direct assessment of gene function by directly altering its expression.
                \begin{itemize}
                    \item \textbf{Gene deletion}: aka gene knockout; breaking or disurbing function of an animal's gene to interfere with proteins, creating deficient or inferior phenotypic traits.
                    \begin{itemize}
                        \item \textbf{Forced overexpression}: inverse of gene deletion; experimentally increasing synthesis of the mRNA.
                        \item \textbf{Compensation}: phenotypic alterations of that tend to make up for the manipulation done by forced expression or gene deletion.
                    \end{itemize}
                    \item \textbf{RNA interference (RNAi)}: allows specific mRNA targets to be silenced in animals with \textit{normal} genomes.
                        \begin{itemize}
                            \item \textbf{Normal genomes}: wild type that is not artificially manipulated.
                            \item RNAi is reversible.
                        \end{itemize}
                    \item \textbf{CRISPR/Cas}: used to edit nuclear DNA in eukaryotic cells.
                        \begin{itemize}
                            \item Can be used to insert sequences that then can be transcribed and tranlasted, providing insights on protein function.
                        \end{itemize}
                \end{itemize}
        \end{itemize}
\end{itemize}

\subsection{Proteomics}
\begin{itemize}
    \item \textbf{Proteomics}: the study of proteins being synthesized by cells and tissues.
        \begin{itemize}
            \item Implies simultaneous study of large numbers of proteins.
            \item Predicting proteins from gene transcription is still very difficult; transcription, translation, and post-translational processing are all regulated dynamically and independently.
        \end{itemize}
    \item \textbf{Two-dimensional gel electrophoresis}: the primary proteomics method that separates complex mixtures of samples using two different protein properties.
        \begin{itemize}
            \item Separated by {\color{o-Sun}isoelectric points} and then {\color{o-Sun}molecular weights}.
        \end{itemize}
\end{itemize}

\subsection{Metabolomics}
\begin{itemize}
    \item \textbf{Metabolomics}: study of organic compounds in the cells and tissues other than macromolecules coded by the genome.
        \begin{itemize}
            \item \textbf{Metabolites}: compounds currently being processed by metabolism and the majority of metabolmics focus of study.
                \begin{itemize}
                    \item e.g. sugars, amino acids, and fatty acids.
                \end{itemize}
        \end{itemize}
    \item \textbf{Nuclear magnetic resonance (NMR)}: primary method of metabolomics that is capable of detecting and quantifying a large variety of compounds through identification of unique signatures in the NMR spectrum.
\end{itemize}
%\endgroup
%%%%%%%%%%%%%%%%%%%%%%%%%%%%% Chapter 3 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%%%%%%%%%%%%%%%%%%%%%%%%%%%%% Chapter 4 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%\begingroup
\clearpage
\section{Physiological Development}\phantomsection
\subsection{Epigenetics}
\begin{itemize}
    \item \textbf{Epigenetics}: modifications in gene expression with {\color{o-Sun}no change in DNA sequence} that are transmitted when genes replicate.
    \item \textbf{Marked}: aka tagged; when DNA is modified in way to alter expression.
        \subsubsection{Mechanisms of Epigenetic Marking}
        \begin{itemize}
            \item \textbf{DNA methylation}: addition of methyl groups to cytosine residues in DNA.
                \begin{itemize}
                    \item Generally represses of silences the gene.
                    \item \textbf{DNA methyltransferase 1 (DNMT1)}: an enzyme acts to perpetuate the pattern of methylation in daughter cells.
                    \item \textbf{Methylome}: the set of all methylated sites.
                \end{itemize}
            \item \textbf{Histone modification}: modified histones that that can make DNA more or less accessbile for transcription.
                \begin{itemize}
                    \item Can be modified by methylation, acetylation, phosphorylation, or other covalent bonding of chemical groups at specific sites.
                    \item Also has mechanisms for perpetuation, e.g. small RNA molecules play a role.
                \end{itemize}
        \end{itemize}
    \item \textbf{Epigenome}: the global summary of marks or a set of epigenetic marks in a cell.
        \subsubsection{Epigenetic Inheritance}
        \begin{itemize}
            \item \textbf{Mitotic inheritance}: aka somatic; perpetuation of marks during the process of cell division by mitosis within an {\color{o-Sun}individual}.
            \item \textbf{Meiotic inheritance}: aka transgenerational; perpetuation of marks during meiosis that results in passing of marks to {\color{o-Sun}offspring}.
        \end{itemize}
    \item Research is continuing to provide strong evidence that epigenetics can radically alter physiology.
    \item Epigenetic marking may also play large roles in lifelong effects due early-life and prenatal environments.
\end{itemize}
%\endgroup
%%%%%%%%%%%%%%%%%%%%%%%%%%%%% Chapter 4 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%%%%%%%%%%%%%%%%%%%%%%%%%%%%% Chapter 5 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%\begingroup
\clearpage
\fancyhead[L]{Week 3}
\section{Transport of Solutes and Water}\phantomsection
\subsection{Passive Transport}
\begin{itemize}
    \item \textbf{Equilibrium}: the state at which a of minimum capacity to do work under locally prevailing conditions.
        \begin{itemize}
            \item A change toward equilibrium is always in the direction of decreasing work potential.
        \end{itemize}
    \end{itemize}
    \subsubsection{Concentration gradients}
        \begin{itemize}
            \item General definition: the difference in concentration between two solutions or regions.
            \item More accurately: \(\dfrac{C_1 - C_2}{X}\) where \(X\) is the distance separating (boundary layer) the regions of concentration of solute particles, making it a colligative property.
            \item \textbf{Fick diffusion equation}: \(J=D\dfrac{C_1 - C_2}{X}\)
                \begin{itemize}
                    \item \(J\) is the net number of solute molecules passing into the low-concentration region from the high-concentration of solute particles, making it a colligative property.
                    \item \textbf{Diffusion coefficient (\textit{D})}: proportionality factor determined by the permeability of the membrane or epithelium as well as the temperature.
                \end{itemize}
            \item Each solute diffuses according to its own concentration of solute particles.
            \item \textbf{Simple diffusion}: aka diffusion; moves solute from an area of high solute concentration to an area of low solution concentration.
            \begin{itemize}
                \item Does not use energy as it can only move material in the direction of the concentration gradient and towards 
                equilibrium.
            \end{itemize}
        \end{itemize}
    \subsubsection{Electrical gradients}
    \begin{itemize}
        \item \textbf{Electrical gradient}: difference in charge across a membrane.
        \item Many solutes bear electrical charge that affects the diffusion of such solutes.
        \item \textbf{Bulk solution}: solution not in contact with with a membrane.
            \begin{itemize}
                \item Has a net charge of zero, this regions do not differe in charge.
                \item Lack of net charge does not affect diffusion in the bulk solution, though does affect diffusion across the cell membranes of epithelia.
                \item \textit{Bulk flow}: physical kinetic movement of fluid, typically due to pressure.
            \end{itemize}
        \item \textbf{Electrochemical gradient}: gradient consisting of the chemical gradient (concentration gradient) and the electrical gradient.   
    \end{itemize}
    \subsubsection{Biological Aspects of Diffusion}
    \begin{itemize}
        \item \textbf{Ion channels}: integral membrane protein that permits the passive transport of inorganic ions by diffusion through the membrane.
            \begin{itemize}
                \item Some can be selective for certain ions, such as {\color{pos}\ch{Na^+}}, {\color{neg}\ch{Cl^-}}, and {\color{pos}\ch{K^+}}
                \item Even the least selective discriminate between {\color{neg}anions} and {\color{pos}cations}
                \item \textbf{Gated channels}: ion channels that can open and close due to the proteins allowing for conformational changes.
                    \begin{itemize}
                        \item \textbf{Voltage-gated}: responds to voltage change.
                        \item \textbf{Stretch-gated}: aka tension gated: responds to physical tensions.
                        \item \textbf{Phosphorylation-gated}: responds due to changes in protein phosphorylation.
                        \item \textbf{Ligan-gated}: responds due to extracelluar signaling.
                    \end{itemize}
            \end{itemize}
        \item \textbf{Permeability}: the ease at which the solute can move through the membrane by diffusion.
            \begin{itemize}
                \item Changed by use and quantity of ion channels 
            \end{itemize}
        \item \textbf{Faciliated diffusion}: the process of spontaneous passive transport of molecules of ions across a biological membrane via transmembrane integral proteins.
        \begin{itemize}
            \item Always occurs in the direction of electrochemical equilibrium.
            \item Solutes are transported faster than they are in simple diffusion.
            \item Solutes must bind reversibly with biding sites on transporter proteins.
            \item Temperature dependence is substantially different due to presence of an activated binding event.
        \end{itemize}
\end{itemize}

\subsection{Active Transport}
\begin{itemize}
    \item \textbf{Active Transport}: the movement of molecules across a cell membrane that is {\color{o-Sun}against} the concentration gradient.
    \item \textbf{Primary active transport}: uses protein pumps that normally use ATP.
        \begin{itemize}
            \item Often transports metal ions such as {\color{pos}\ch{Na^+}}, {\color{pos}\ch{K^+}}, {\color{pos}\ch{Mg^2+}}, and {\color{pos}\ch{Ca^2+}}
            \item Most enzymes used are transmembrane ATPases, such as the sodium-potassium pump, which moves three {\color{pos}\ch{Na^+}} ions out of the cell for every two {\color{pos}\ch{K^+}} moved into the cell.
        \end{itemize}
    \item \textbf{Secondary active transport}: uses potential energy derived through movement of ions (using transporter proteins and ATP) across the electrochemical gradient.
        \begin{itemize}
            \item \textbf{Antiporter}: one substrate is transported across the membrane while the other is contransported in the opposite direction.
            \item \textbf{Symporter}: two substrates are transported in the same direction across the membrane.
            \item {\color{pos}\ch{Na^+}}, {\color{pos}\ch{K^+}}, or {\color{pos}\ch{H^+}} ions are usually the ones moving down the gradient and used to transport the desired ion up the relative gradient. 
        \end{itemize}
\end{itemize}

\subsection{Diversity and Modulation of Channels and Transporters}
\begin{itemize}
    \item \textbf{Multiple molecular forms}: many forms of a channel and transporter proteins are common. 
        \begin{itemize}
            \item Different species have evolved differnet molecular forms, which can modulate function and efficiency.
            \item Allows for opportunities for adaptation.
        \end{itemize}
    \item \textbf{Modulation by gene expression}: common channels and transporters can be modulated throughout a lifetime via gene expression responses to environmental circumstances.
    \item \textbf{Noncovalent and covalent modulation}: both ligand (often noncovalent) binding and phosphorylation (covalent) allow for {\color{o-Sun}rapid} regulation of channels and transporters.
    \item \textbf{Insertion-and-retrieval modulation}: the {\color{o-Sun}location} of proteins in the membrane allow for another way of regulating activity.
        \begin{itemize}
            \item Some proteins are held in reserve, and {\color{o-Sun}inserted} into the membrane when necessary.
            \item Inverse is also true, some proteins can be {\color{o-Sun}retrieved} from the membrane in order to modulate usage.
            \item Often only takes minutes for modulation to occur.
        \end{itemize}
\end{itemize}

\subsection{Colligative Properties of Aqueous Solutions}
\begin{itemize}
    \item \textbf{Colligative properties}: the properties of solutions that depend on the {\color{o-Sun}ratio} between solute particles and solevent molecules.
        \begin{itemize}
            \item Not dependent on the nature of the chemical species present.
            \item Effects include: relative lowering of vapour pressure, elevation of boiling point, depression of freezing point, and \textit{osmotic pressure}.
        \end{itemize}
    \item \textbf{Vapour pressure}: the pressure of the vapour which is in equilibrium with that liquid.
        \begin{itemize}
            \item Vapour pressure of a solvent is lowered when a non-volatile solute is dissolved in it to form a solution.
        \end{itemize}
    \item \textbf{Boiling and freezing points}: additions of solute help stabilize the solvent in the liquid phase, lowering chemical potential, and thus a lower tendency to move to gas phase or solid.
        \begin{itemize}
            \item \textbf{Freezing point depression}: lowering of freezing point of a solvent with the addition of a solute that is insoluble in the solid solvent.
            \item \textbf{Boiling point elevation}: increased by the by the addition of a non-volatile solute.
            \item Both are proportional to the lowering of vapour pressure in a dilute solution. 
        \end{itemize}
\end{itemize}

\subsection{Osmosis}
\begin{itemize}
    \item \textbf{Osmosis}: the spontaneous net movement of solvent molecules through a selectively permeable membrane into a region of higher solute concentration.
        \begin{itemize}
            \item Can be made to do work.
            \item The primary means by which {\color{o-Sun}water} is transported into and out of cells.
            \item \textbf{Turgor}: the force and which the cell pushes the plasma membrane against the cell wall.
            \item Turgor is largely mantained by osmosis across the cell membrane between the interior and its relatively hypotonic environment.
        \end{itemize}
    \item \textbf{Osmotic pressure}: the external pressure required to be applied so that there is no net movement of a solvent across the membrane.
        \begin{itemize}
            \item The semipermeable membrane allows the passage of solvent molecules but not the solute particles.
            \item Also defined as the measure of tendency of a solution to take in pure solvent by osmosis. {\color{G-Moon}"Water wants to go where solutes are"}
            \item \textbf{Osmotic gradient}: the difference in pressure between the solution and the pure liquid solvent when the two are in equilibrium across a semipermeable membrane.
                \begin{itemize}
                    \item Formula: \(K\dfrac{\Pi_1 - \Pi_2}{X}\)
                    \item i.e., the {\color{o-Sun}rate} at which water crosses the membrane by osmosis.
                    \item Similar to the Fick equation for concentration gradient, except $\Pi_{1\&2}$ are the osmotic pressures of the solutions on each side of the membrane, and \(K\) is the osmotic permeability of the membrane + temperature.
                \end{itemize}
            \item Proportional to the concentration of solute particles, making it a colligative property.
            \item \textbf{Isosmotic}: when two solutions have the same osmotic pressure.
            \item When solution \(A < B\) in terms of osmotic pressure then:
                \begin{itemize}
                    \item A is \textbf{hyposmotic} to B --- A has less solutes than B
                    \item B is \textbf{hyperosmotic} to A --- B has more solutes than A.
                    \item The direction of net water movement by osmosis is from hyposmotic solution into the hyperosmotic one, i.e., \(A \rightarrow B\)
                \end{itemize}
        \end{itemize}
    \item Water is still capable of diffusing directly thorugh lipid membranes.
    \item \textbf{Aquaporins}: water-channel proteins that greatly increase water transport.
        \begin{itemize}
            \item Water transport through aquaporins is strictly passive.
        \end{itemize}
    \item Nonpermeating solutes often create persistent osmotic-gradient componenets across semipermable membranes.
        \begin{itemize}
            \item Plays a important role in blood, as blood pressure forces water out, but proteins create persistent tendency to take up water; termed {\color{o-Sun}colloid osmotic pressure} of the blood.
        \end{itemize}
    \item Passive solute transport and osmosis interact. 
        \begin{itemize}
            \item \textbf{Solvent drag}: when solute moves with water crossing the membrane.
            \item Tends to alter electrochemical gradients which plays a continuous role in rates of passive transport of both water and solutes.
        \end{itemize}
    \item Active solute transport provides a mean to control passive water transport.
        \begin{itemize}
            \item Water transport is strictly passive, though control of solutes indirectly allows for metabolic water transport.
        \end{itemize}
\end{itemize}

\subsection{Osmoregulation}
{\color{darklc}Excerpt from Chapter 27: Water and Salt in Physiology $\mapsto$}
\begin{itemize}
    \item \textbf{Osmoregulation}: the active regulation of the osmotic pressure of an organism's body fluids.
        \begin{itemize}
            \item Detected by osmoreceptors, primarily found in the hypothalamus. 
            \item Acts to maintain homeostasis of the water content and electrolyte concentration.
        \end{itemize}
    \item \textbf{Osmoconformers}: match their body osmolarity to their environment actively or passively. 
        \begin{itemize}
            \item Most marine invertebrates are osmoconformers.
        \end{itemize}
    \item \textbf{Osmoregulators}: tightly regulate their body osmolarity through internal conditions. 
        \begin{itemize}
            \item More common in animals.
        \end{itemize}
    \item \textbf{Volume conformity}: passive changes in body-fluid volume.
    \item \textbf{Volume regulation}: regulation of the {\color{o-Sun}total} amount of water in a body fluid.
    \item There is also ionic regulation and conformity that are subject to ion-specific physiological controls. 
    \item Influx of \ch{H2O} will tend to lower osmotic pressure, dilute ions, and increase volume.
\end{itemize}
%\endgroup
%%%%%%%%%%%%%%%%%%%%%%%%%%%%% Chapter 5 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%%%%%%%%%%%%%%%%%%%%%%%%%%%%% Chapter 7 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%\begingroup
\clearpage
\setcounter{section}{6}
\section{Nutrition, Feeding, and Digestion}\phantomsection
\subsection{}
\begin{itemize}
    \item 
\end{itemize}
%\endgroup
%%%%%%%%%%%%%%%%%%%%%%%%%%%%% Chapter 7 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%


\clearpage
\fancyhead[L]{Midterm II}
%\begingroup

%\endgroup
\end{document}